tests/containers/docker_image.pm

```diff
@@ -23,19 +24,21 @@ use Mojo::Base qw(consoletest);
 use testapi;
 use utils;
 use containers::common;
+use containers::utils;
 use containers::container_images;
 use containers::urls 'get_suse_container_urls';
 use version_utils qw(get_os_release check_os_release);
 
 sub run {
+    my ($self) = @_;
     my ($image_names, $stable_names) = get_suse_container_urls();
     my ($running_version, $sp, $host_distri) = get_os_release;
     my $runtime = "docker";
 
     install_docker_when_needed($host_distri);
     allow_selected_insecure_registries(runtime => $runtime);
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
-
+    check_docker_firewall() if (check_runtime_version(get_docker_version(), ">=20.10") && $self->firewall() eq 'firewalld' && $host_distri ne 'centos' && $host_distri ne 'ubuntu');
```

Comment:
Why is this in both `docker.pm` as well as herer in `docker_image.pm`?

Reply:
i answered on this [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12448#discussion_r625141156)